lib/config/presets/internal/group.ts

```diff
@@ -116,7 +116,11 @@ const staticGroups = {
     packageRules: [
       {
         groupName: 'Drupal core',
-        matchPackageNames: ['drupal/core', 'drupal/core-*'],
+        matchPackageNames: [
+          'drupal/core',
+          'drupal/core-*',
+          'pantheon-systems/drupal-integrations'
```

Comment:
Doesn't seem to have the _exact_ same release cadence, e.g. its latest is 11.2.0 now and not 11.2.2. This could result in "immortal" PRs from Renovate due to the mismatch of to-versions, but maybe ok